cinema/admin.py

```diff
@@ -0,0 +1,3 @@
+from django.contrib import admin
+
+# Register your models here.
```

Comment:
Consider registering your models here if you want them to be manageable through the Django admin interface. You can do this by importing your models and using `admin.site.register(ModelName)`.